pylearn2/scripts/papers/maxout/compute_test_err.py

```diff
@@ -51,8 +51,6 @@ def accs():
                             data_specs = model.cost_from_X_data_specs())
     for item in iterator:
         x_arg, y_arg = item
-        if Xb.ndim > 2:
-            x_arg = test.get_topological_view(x_arg)
```

Comment:
@bouthilx Actually, in the past, the iterator was not used, and the script looped directly on `test.X`: https://github.com/lisa-lab/pylearn2/commit/ff954c7599f83157d9caba13bf39989ae7ae803c
`FiniteDatasetIterator` will create the appropriate `convert` functions for the sources where the `Space` requested is different from the "native" one: https://github.com/lisa-lab/pylearn2/blob/master/pylearn2/utils/iteration.py#L926
These functions will then be called on each minibatch by the iterator's `next()` method.